pgmoon/json.moon

```diff
@@ -0,0 +1,10 @@
+encode_json = (tbl) ->
+  json = require "cjson"
+  enc = json.encode tbl
+  "'#{(enc\gsub "'", "''")}'"
```

Comment:
Indeed. But I like it better if it reuses the function too, I should have checked the array module.